lib/jekyll/document.rb

```diff
@@ -1,15 +1,16 @@
 # encoding: UTF-8
 
 module Jekyll
+  # rubocop:disable ClassLength
```

Comment:
I couldn't think of a way to reduce the size of this class (it's 275 lines now, max is 240) 😞 
Any ideas?)